File: app/src/main/java/net/xzos/UpgradeAll/viewmodels/ItemCardView.java
package net.xzos.UpgradeAll.viewmodels;

public class ItemCardView {
    private int databaseId;
    private String name;
    private String desc;
    private String api;

    public ItemCardView(int databaseId, String name, String desc, String api) {
        this.databaseId = databaseId;
        this.name = name;
        this.desc = desc;
        this.api = api;
    }

    public int getDatabaseId() {
        return databaseId;
    }

    public String getName() {
        return name;
    }

    public String getDesc() {
        return desc;
    }

    public String getApi() {
        return api;
    }


}

File: app/src/main/kotlin/net/xzos/upgradeAll/ui/viewmodels/view/ItemCardView.kt
package net.xzos.upgradeAll.ui.viewmodels.view

import net.xzos.upgradeAll.server.app.manager.module.App


class ItemCardView internal constructor(
        val name: String? = null,
        val desc: String? = null,
        val extraData: ItemCardViewExtraData = ItemCardViewExtraData()
)

data class ItemCardViewExtraData(
        val app: App? = null,
        val uuid: String? = null
)
